_extensions/wjschne/apaquarto/_apa_title.qmd

```diff
@@ -27,7 +27,9 @@ if (is_pdf) {
       
     }
     
+    opts_current$lock(FALSE)
     knitr::opts_current$set(output = "asis", fig.env = NULL)
+    opts_current$lock(TRUE)
```

Comment:
Just curious about the reason to modify `opts_current`. This is to help us cover all usage. 

Do I understand correctly that you want to set / modify some options from within a knit hook ? 
So that other part of the knitr code after the book is ran can behave differently ?

Reply:
@wjschne Can you answer this?

Reply:
I don't remember exactly why I had to do this, but it was necessary to avoid an error caused when using flextable.

Reply:
Ok thank you.

Reply:
I think you are trying to solve something related to Quarto and Flextable interaction. 

I don't think you should have to handle that so probably an issue to fix somewhere. 

Anyhow, usually to modify options, the construct to use is Options Hook (https://bookdown.org/yihui/rmarkdown-cookbook/option-hooks.html)

Here is seems that you want to be sure that `fig.env` is NULL. You could use an option hook for that, by checking a condition. See example at https://github.com/yihui/knitr/issues/1798#issuecomment-1730303063

Anyhow, current solution to unlock / lock works, but I believe this is still a hack and not the expected way to do that.